### ISSUE TITLE
Translation for pt_PT

### ISSUE THREAD

**Issue by FlavioCFOliveira** [go-playground/validator#611]
Hello,

How can i help to translate the validation messages for pt_PT?

Should i fork and make a pull request?

**Comment by deankarn** [comment]
Yes the best way would be to fork and create a PR @FlavioCFOliveira 

glad you're interested in contributing :)

**Comment by FlavioCFOliveira** [comment]
Ok, i've sent the PR
